Source/ACE.Server/Command/Handlers/PlayerCommands.cs

```diff
@@ -48,8 +54,9 @@ public static void HandleQuests(Session session, params string[] parameters)
                     Console.WriteLine($"Couldn't find quest {playerQuest.QuestName}");
                     continue;
                 }
+                var minDelta = (uint)(quest.MinDelta * PropertyManager.GetDouble("quest_mindelta_rate", 1).Item);
```

Comment:
```suggestion
                var minDelta = (uint)(quest.MinDelta * PropertyManager.GetDouble("quest_mindelta_rate").Item);
```i think we generally avoid fallback default as that had led to some instances of property desync inbetween saves...